ISPN-13972: Update property realm note in Security realms doc

dfitzmau opened the pull request:
JIRA: https://issues.redhat.com/browse/ISPN-13972

dfitzmau commented:
Hello @wfink .

Would you be OK to review this PR?

This PR stems from the following Jira: https://issues.redhat.com/browse/ISPN-13972

wfink reviewed (approved):
I think the simplified explanation is good enough

dfitzmau commented:
Hi @dvagnero and @pruivo .

Would you both be OK to review this PR that stems from https://issues.redhat.com/browse/ISPN-13972

dvagnero commented on documentation/src/main/asciidoc/topics/ref_server_realm_property.adoc:
```diff
@@ -8,7 +8,11 @@ Passwords can be pre-digested with the `DIGEST-MD5` and `DIGEST` authentication
 ====
-Properties files contain headers that associate them with security realms in {brandname} Server configuration.
+You can avoid authentication issues that relate to a property file by using the {brandname} CLI to enter the correct security realm name to the file. 
+
```
Please remove the empty lines. In general place each sentence on its own line but don't leave any empty lines between.

dvagnero replied:
This will go to downstream docs once it's merged (the attributes will be substituted from downstream file).
Regarding starting each sentence on its won line - this applies to all our docs. You can check few adoc files and see the format

dvagnero commented on documentation/src/main/asciidoc/topics/ref_server_realm_property.adoc:
```diff
@@ -8,7 +8,11 @@ Passwords can be pre-digested with the `DIGEST-MD5` and `DIGEST` authentication
+
+You can find the correct security realm name of your {brandname} server by opening the `infinispan.xml` file and navigating to the `<security-realm name>` property. 
+
+If you decide to copy a property file from one {brandname} server to another, you must ensure that the security realm name appropriates to the correct authentication mechanism for the target endpoint. 
```
I think it can be little shorter, for example
```suggestion
When you copy a property file from one {brandname} Server to another, make sure that the security realm name appropriates to the correct authentication mechanism for the target endpoint. 
```

dfitzmau replied:
Updated.

dvagnero reviewed (changes requested):
Thanks @dfitzmau  I added few comments

dfitzmau commented:
Hello @wfink .

Thank you for reviewing this PR. 

Just to clarify that the updates requested from https://issues.redhat.com/browse/ISPN-13972, target the 13.0.11 release? Do I need to back port these changes post-merge of this PR?

dfitzmau commented:
> Thanks @dfitzmau I added few comments

Thanks,@[dvagnero](https://github.com/dvagnero) . I incorporated your feedback. Please let me know your thoughts.
